components/org.wso2.finance.open.banking.conformance.mgt/src/main/java/org/wso2/finance/open/banking/conformance/mgt/dao/impl/UserDAOImpl.java

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.finance.open.banking.conformance.mgt.dao.impl;
+
+import org.wso2.finance.open.banking.conformance.mgt.dao.UserDAO;
+import org.wso2.finance.open.banking.conformance.mgt.db.DBConnector;
+import org.wso2.finance.open.banking.conformance.mgt.db.SQLConstants;
+import org.wso2.finance.open.banking.conformance.mgt.dto.UserDTO;
+
+import java.sql.*;
+
```

Comment:
same empty code blocks and improper resource closing issues can be seen here too.